packages/synapse-react-client/src/lib/containers/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm.tsx

```diff
@@ -305,28 +183,6 @@ export default function RequestDataAccessStep2(
     )
   }
 
-  const getSubmissionMsg = (submission_resp: ACTSubmissionStatus) => {
```

Comment:
Was only used in the case where the submission would be immediately rejected, so it can be deleted.